### ISSUE TITLE
Tests fail under GAP master branch

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/datastructures#123]
Please see e.g. https://travis-ci.org/gap-infra/gap-docker-pkg-tests-master/jobs/597455020

```
########> Diff in /home/gap/inst/gap-master/pkg/datastructures-0.2.4/tst/hashf\
unctions/recursive.tst:56
# Input is:
HashBasic(SymmetricGroup(3));
# Expected output:
Error, Unable to hash object (component)
# But found:
Error, Unable to hash component object
########
       4 ms (0 ms GC) and 383KB allocated for hashfunctions/recursive.tst
testing: /home/gap/inst/gap-master/pkg/datastructures-0.2.4/tst/hashfunctions/\
string.tst
      99 ms (3 ms GC) and 17.0MB allocated for hashfunctions/string.tst
testing: /home/gap/inst/gap-master/pkg/datastructures-0.2.4/tst/hashfunctions/\
trans.tst
    1043 ms (913 ms GC) and 340MB allocated for hashfunctions/trans.tst
testing: /home/gap/inst/gap-master/pkg/datastructures-0.2.4/tst/hashmap.tst
########> Diff in /home/gap/inst/gap-master/pkg/datastructures-0.2.4/tst/hashm\
ap.tst:238
# Input is:
DS_Hash_Value(infinity, 1);
# Expected output:
Error, <ht> must be a hashmap object (not a object (positional))
# But found:
Error, <ht> must be a hashmap object (not a positional object)
########
```

**Comment by fingolfin** [comment]
Yeah, that's due to the changed output of `TNAM_OBJ`. Completely harmless of course, but tests should not fail of course. Will fix.

**Comment by fingolfin** [comment]
Resolved by PR #127
